docs/csharp/fundamentals/object-oriented/polymorphism.md

```diff
@@ -32,7 +32,7 @@ In C#, every type is polymorphic because all types, including user-defined types
 
 ### Virtual members
 
-When a derived class inherits from a base class, it gains all the methods, fields, properties, and events of the base class. The designer of the derived class has different choices for the behavior of virtual methods:
+When a derived class inherits from a base class, it gains all the virtual methods, virtual properties, and virtual events of the base class. The designer of the derived class has different choices for the behavior of virtual methods:
```

Comment:
This paragraph was making a different point. I agree that the previous text was confusing. How about this instead:

```suggestion
When a derived class inherits from a base class, it includes all the members of the base class. All the behavior declared in the base class is part of the derived class. That enables objects of the derived class to be treated as objects of the base class. Access modifiers (`public`, `protected`, `private` and so on) determine if those members are accessible from the derived class implementation. Virtual methods gives the designer different choices for the behavior of the derived class:
```

Reply:
I have amended the PR with new wording. I've made slight change to your suggested wording. Please review it again